src/llmcompressor/transformers/finetune/data/base.py

```diff
@@ -97,6 +97,7 @@ def __call__(self, add_labels: bool = True) -> DatasetType:
                 self.preprocess,
                 batched=False,
                 num_proc=self.data_args.preprocessing_num_workers,
+                load_from_cache_file=not self.data_args.overwrite_cache,
```

Comment:
Why did we change this?

Reply:
The `load_from_cache_file` argument determines if the `datasets.map` function caches its results. SparseML neglected to add this argument to the preprocess function, and I didn't bother fixing it during the VLM dataset changes.